hmf/density_field/transfer_models.py

```diff
@@ -20,6 +20,11 @@
 except ImportError:
     HAVE_CAMB = False
 
+try:
+    import classy
+except ImportError as e:
+    pass
```

Comment:
Set `HAVE_CLASS=False` here if it fails, then raise an error in the class `__init__` if it's false.